files/docker-container-logrotate

```diff
@@ -0,0 +1,9 @@
+/var/lib/docker/containers/*/*.log {
+  rotate 0
+  hourly
+  notifempty
+  nocompress
+  size 20k
```

Comment:
Ideally on non coreos or Redhat distros you could configure the logroatated cron to run on an hourly basis, I couldn't find a way to do that on coreos, still need to dig deep.
Idea for hourly is to rotate the logs hourly.